btc-lsp/src/BtcLsp/Data/Orphan.hs

```diff
@@ -55,3 +55,24 @@ instance Out SomeException where
     PP.text . Universum.show
   doc =
     docPrec 0
+
+tr :: Btc.TransactionID -> ByteString
+tr x = fromString $ unpack x
+
+tx :: Integer -> Word32
+tx = fromIntegral
```

Comment:
This is unsafe because Integer > Word32
User TryFrom instance from Witch instead